src/components/cylc/analysis/BoxPlot.vue

```diff
@@ -152,71 +222,6 @@ export default {
         { title: `Max ${this.timingOption} time`, value: `max${upperFirst(this.timingOption)}Time` },
       ]
     },
-
-    chartOptions () {
-      return {
-        chart: {
-          animations: {
-            enabled: useReducedAnimation().value ? false : this.animate,
```

Comment:
Do we really have to shift this whole block just to handle this one usage?

Reply:
Yes

Reply:
Dang.